apps/galleries_institutions/components/filter_facet/locationSynonyms.coffee

```diff
@@ -0,0 +1,4 @@
+module.exports = [
+  ['st', 'saint']
```

Comment:
Let's add to this list sparingly, since there is a performance hit — just barely perceptible when you compare the location filter to the other two — while doing the typeahead.